Test_Task/main.rb

```diff
@@ -0,0 +1,59 @@
+require 'watir'
+require 'nokogiri'
+require 'json'
+require_relative './transactions'
+
+class WebBanking
+  attr_reader :browser, :page
+  def initialize
+    @browser = Watir::Browser.new :chrome
+  end
+
+  def collect_info
+    sign_in
+    go_to_info
+    tr = Transactions.new
+    tr.go_to_transactions(browser)
+    lol = {
+        name: parse_name,
+        balance: parse_balance,
+        currency: parse_currency,
+        nature: parse_nature,
+        transactions: tr.get_transaction_info(browser)
+    }
+    puts JSON.pretty_generate(accounts: [lol])
```

Comment:
hmm, there should be a loop through accounts,
if I run your script under my credentials it will return one account, but I have two :(

anyways, you have to write method `parse_accounts` which will return an array with accounts. 
method `parse_accounts` should call method `parse_account` which call parsers and create a hash with all account data + transactions.